netsuite/src/TypeScript/matching-handler.ts

```diff
@@ -0,0 +1,55 @@
+import {createMatchedLineFromUnmatched, existInMatchedLines} from "./models/matched-line/matched-line"
+import {UnmatchedLine} from "./models/unmatched/unmatched-constants"
+import {toInteger} from "./utils"
+import {createMatchedGroup} from "./models/match-group/match-group"
+import log from 'N/log'
+import {deleteUnmatchedLineById} from "./models/unmatched/index"
+
+
+export type validationResult =
+    | { valid: true }
+    | { valid: false, message: string }
+
+export function handleMatching(unmatchedLines: UnmatchedLine[]):string | number {
```

Comment:
```suggestion
export function createMatchedGroup(unmatchedLines: UnmatchedLine[]): number {
```
Eftersom detta är en REST-endpoint för att skapa upp en resurs med en POST-request anser jag att den borde kallas createMatch eller snarare createMatchedGroup.

Jag tycker sedan att vi generellt borde försöka undvika att returnera både string och number när vi genomför NetSuite operationer, och välja en av dem. I detta fall vore det fint med number.

edit: Såg att det heter matched group nästan överallt och jag råkade döpa den till matching group.

Reply:
NS returnerar ju string by default, tycker du alltså vi ska hålla på parsa alla resultat som är id:n som fås tillbaka från NS?

Reply:
Matched lines och matched group skapas och unmatched lines raderas. Så createMatchedGroup tänker jag blir vilseledande men kan ändra till createMatch